api/users.js

```diff
@@ -2,12 +2,21 @@ const express = require("express");
 
 const userController = require("../controllers/users");
 const { auth } = require("../middlewares/auth");
+const { uploadMiddleware } = require("../middlewares/upload");
 
 const router = express.Router();
 
 router.post("/signup", userController.register);
 router.post("/login", userController.login);
 router.get("/logout", auth, userController.logout);
 router.get("/current", auth, userController.current);
+router.patch("/:userId/subscription", auth, userController.updateSubscription);
+router.patch(
+  "/avatars",
+  auth,
+  uploadMiddleware.single("avatar"),
+  userController.updateAvatar
+);
+router.delete("/", userController.deleteUserByMail);
```

Comment:
Ta ścieżka powinna być chroniona, nawet więcej, tylko admin powinien mieć do niej dostęp. Użytkownik jak sam siebie usunie to momentalnie straci dostęp do aplikacji. To na przyszłość :)  I też pytanie jak usuwasz uzytkownika to pewnie wszystkie jego dane itp.

Reply:
dodane:)